config/application.rb

```diff
@@ -1,6 +1,12 @@
 require File.expand_path('../boot', __FILE__)
 require File.expand_path('../preinitializer', __FILE__)
-require 'rails/all'
+require 'rails'
+require 'active_record/railtie'
+require 'action_controller/railtie'
+require 'action_view/railtie'
+require 'action_mailer/railtie'
+require 'active_job/railtie'
+require 'sprockets/railtie'
```

Comment:
`rails/all` is a still a [current thing](https://github.com/rails/rails/blob/master/railties/lib/rails/all.rb)

The only omitted framework is test_unit, something we don't use (right?). Removing the railtie also removes it from [loading its rake tasks](https://github.com/rails/rails/blob/master/railties/lib/rails/test_unit/railtie.rb#L14-L16). That's `test, test:all, test:db`, that sort of stuff. `test` currently doesn't do anything (because its from test_unit), but I've replaced that with our own `test` on line 32 in test.rake.